{"uuid":"b8212d00-5894-4bc9-ac33-b4f68a1acf16","historyId":"4e36b4c917dc462de4e22e9aab00be3e","testCaseId":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.UserServiceTest]/[method:login__AuthException()]","testCaseName":"Login -- Throws AuthException","fullName":"org.academo.academo.London.service.UserServiceTest.login__AuthException","labels":[{"name":"tag","value":"unit"},{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.UserServiceTest]/[method:login__AuthException()]"},{"name":"host","value":"Rockio"},{"name":"thread","value":"53622@Rockio.main(3)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"org.academo.academo.London.service.UserServiceTest"},{"name":"testClass","value":"org.academo.academo.London.service.UserServiceTest"},{"name":"testMethod","value":"login__AuthException"},{"name":"suite","value":"User Service Tests"}],"links":[],"name":"Login -- Throws AuthException","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1757153225140,"stop":1757153225142}